tensorflow/contrib/learn/python/learn/estimators/head.py

```diff
@@ -35,8 +35,6 @@
 from tensorflow.python.framework import ops
 from tensorflow.python.framework import sparse_tensor
 from tensorflow.python.ops import array_ops
-from tensorflow.python.ops import control_flow_ops
```

Comment:
Oh wait, never mind. I see that control_flow_ops is actually been getting called elsewhere in the code. I must have been looking at the different code file when I ran my search.